package/libs/wolfssl/Config.in

```diff
@@ -49,7 +49,7 @@ config WOLFSSL_HAS_WPAS
 
 config WOLFSSL_HAS_ECC25519
 	bool "Include ECC Curve 25519 support"
-	default n
+	default y
```

Comment:
Do you have the package size increase for the change?  Unless it is negligible, we may want to do:
```
default y if !SMALL_FLASH 
```


Since this is a core package, devs may want to chime in.

Reply:
Would it make more sense to bundle this and `WOLFSSL_ALT_CERT_CHAINS` fix together since both fixes address the same type of issue?